frontend/apps/filemanager/filemanagerbookinfo.lua

```diff
@@ -0,0 +1,186 @@
+--[[--
```

Comment:
This PR already makes it available in Reader (I mentionned that in the commit message).
`filemanagerbookinfo.lua` is similar in many aspects to `apps/filemanager/filemanagerhistory.lua`: it is imported in Reader, uses `registerToMainMenu`/`addToMainMenu` to add itself to Reader's menu.
I think it belongs into `frontend/apps`, it just get data from files, settings, make some data structure out of them (_business_ logic) and call a real generic widget: KeyValuePage (_view_ logic). Just like filemanagerhistory.lua calls a the generic Menu widget.
Putting it into `frontend/ui/widget` would need this code to be splitted into 2 files (one in apps/ for the menu, one in ui/widget/), and I don't see where we would use again the ui/widget part (which would contain some business logic).